org.adempiere.base/src/org/compiere/util/Env.java

```diff
@@ -851,6 +851,24 @@ public static boolean isAutoCommit (Properties ctx, int WindowNo)
 		return isAutoCommit(ctx);
 	}	//	isAutoCommit
 
+	/**
+	 * Is Show Technical Information 
+	 * @param ctx context
+	 * @return true if IsShowTechnicalInfOnHelp on User Preference
+	 */
+	public static boolean IsShowTechnicalInfOnHelp(Properties ctx)

```

Comment:
Please remove the ctx parameter, not needed.

Reply:
this is just for entity type ? perhaps should just use "IsShowEntityType" instead ?